FLT/HIMExperiments/module_topology.lean

```diff
@@ -171,7 +238,7 @@ noncomputable def LinearMap.add (M : Type*) [AddCommGroup M] [Module A M] :
 
 -- Now say we have a non-commutative `A`-algebra `D` which is free of finite type.
 
-variable (D : Type*) [Ring D] [Algebra A D] [Module.Finite A D] [Module.Free A D]
+variable (D : Type*) [Ring D] [Algebra A D]
```

Comment:
I am more nervous about the bilinear map sorry without these instances -- can you put them back, but just above ` Module.continuous_bilinear` ?

Reply:
We'll try to figure it out with these conditions then.